todo_app/templates/index.html

```diff
@@ -15,7 +15,11 @@ <h2>New Item</h2>
         <input class="form-control form-control-lg" type="text" name="title" id="title" placeholder="What do you want to do?" required>
         <label for="description">Description</label>
         <input class="form-control form-control-lg" type="text" name="description" id="description" placeholder="Perhaps a more detailed description?">
-        <button type="submit" class="btn btn-primary">Add</button>
+        {% if current_user.id in WRITER_LIST or not current_user.is_authenticated %} 
```

Comment:
Minor but worth thinking about the UX here - is it better to show the buttons/inputs disabled as opposed to not showing them at all? In some scenarios it's helpful to know that functionality exists but you just don't have permission, but often I think it's better (and arguably a small amount safer) to just hide the controls entirely if they're not relevant to that user

Reply:
Good idea, I'm adding it into my next update